sormas-backend/src/test/java/de/symeda/sormas/backend/caze/CaseClassificationLogicTest.java

```diff
@@ -1049,6 +1050,30 @@ public void ruleOutFalsePositivesForPlague() {
 		assertEquals(CaseClassification.SUSPECT, caze.getCaseClassification());
 	}
 
+	@Test
+	public void testAutomaticClassificationForLuxembourgServerConfirmed() {
+		MockProducer.getProperties().setProperty(ConfigFacadeEjb.COUNTRY_LOCALE, CountryHelper.COUNTRY_CODE_LUXEMBOURG);
+		CaseDataDto caze = creator.createUnclassifiedCase(Disease.CORONAVIRUS);
+		caze = getCaseFacade().save(caze);
+		creator.createPathogenTest(caze, Disease.CORONAVIRUS, PathogenTestType.ANTIBODY_DETECTION, PathogenTestResultType.NEGATIVE);
+		creator.createPathogenTest(caze, Disease.CORONAVIRUS, PathogenTestType.PCR_RT_PCR, PathogenTestResultType.POSITIVE);
+		caze = getCaseFacade().save(caze); //triggers the recalculation of case classification
```

Comment:
there is no need to save again the unchanged case, pathogen test creation should also trigger the classification

Reply:
removed